nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -321,6 +321,22 @@ in
         session   include       login
       '';
 
+      login.fprintAuth = mkIf config.services.fprintd.enable false;
+      gdm-fingerprint.text = mkIf config.services.fprintd.enable ''
+        auth       required                    pam_shells.so
+        auth       requisite                   pam_nologin.so
+        auth       requisite                   pam_faillock.so      preauth
+        auth       required                    ${pkgs.fprintd}/lib/security/pam_fprintd.so
+        auth       optional                    pam_permit.so
+        auth       required                    pam_env.so
+        auth       [success=ok default=1]      ${pkgs.gnome.gdm}/lib/security/pam_gdm.so
```

Comment:
I am seeing the following in logs:

gdm-fingerprint][1033624]: PAM bad jump in stack

Probably caused by the `default=1`, which should skip one `auth` instruction when `pam_gdm` gives anything but success, there are no further `auth`instructions.

Reply:
`security.pam.services.gdm-fingerprint.enableGnomeKeyring = true;` does not appear to do anything for me.

But I am now pretty sure invoking it manually as in https://github.com/NixOS/nixpkgs/commit/b3e54964bee0f0c87ad1e57558de2d1fbb3147b5 would be fine.

If we want to make it conditional, we would need to replace the one after the jump with the `pam_permit.so` line when disabled.